test/metamorpho/TestMetaMorphoSnippets.sol

```diff
@@ -223,30 +263,17 @@ contract TestMetaMorphoSnippets is IntegrationTest {
         vault.deposit(secondDeposit, ONBEHALF);
         vm.stopPrank();
 
-        collateralToken.setBalance(BORROWER, 2 * MAX_TEST_ASSETS);
-        vm.startPrank(BORROWER);
-        morpho.supplyCollateral(allMarkets[0], MAX_TEST_ASSETS, BORROWER, hex"");
-        morpho.borrow(allMarkets[0], firstBorrow, 0, BORROWER, BORROWER);
-
-        morpho.supplyCollateral(allMarkets[1], MAX_TEST_ASSETS, BORROWER, hex"");
-        morpho.borrow(allMarkets[1], secondBorrow / 4, 0, BORROWER, BORROWER);
-        vm.stopPrank();
-
-        Id id0 = Id(allMarkets[0].id());
-        Id id1 = Id(allMarkets[1].id());
-
-        Market memory market0 = morpho.market(id0);
-        Market memory market1 = morpho.market(id1);
-
-        uint256 rateMarket0 = snippets.supplyAPYMarket(allMarkets[0], market0);
-        uint256 rateMarket1 = snippets.supplyAPYMarket(allMarkets[1], market1);
+        uint256 rateMarket0 = snippets.supplyAPYMarket(marketParams0, market0);
+        uint256 rateMarket1 = snippets.supplyAPYMarket(marketParams1, market1);
         uint256 avgApyNum = rateMarket0.wMulDown(firstDeposit) + rateMarket1.wMulDown(secondDeposit);
 
         uint256 expectedAvgApy = avgApyNum.mulDivDown(WAD - vault.fee(), firstDeposit + secondDeposit);
 
         uint256 avgSupplyApySnippets = snippets.supplyAPYVault(address(vault));
 
-        assertEq(avgSupplyApySnippets, expectedAvgApy, "avgSupplyApySnippets == 0");
+        assertGt(rateMarket0, 0, "avgSupplyApySnippets == 0");
+        assertGt(rateMarket1, 0, "avgSupplyApySnippets == 0");
+        assertApproxEqAbs(avgSupplyApySnippets, expectedAvgApy, MIN_TEST_ASSETS);
```

Comment:
The test doesn't pass if you reduce the margin ?

Reply:
it is 1e8 -> Passing
1e7 -> breaking